frontend/packages/pipelines-plugin/src/utils/pipeline-utils.ts

```diff
@@ -457,3 +458,17 @@ export const pipelinesTab = (kindObj: K8sKind) => {
       return null;
   }
 };
+
+type ResultColumn = [string, string];
+
+export type ResultCells = {
+  rows: ResultColumn[];
+  columns: ResultColumn;
+};
+
+export const getCellsFromResults = (results: TektonResultsRun[]): ResultCells => {
+  return {
+    columns: ['pipelines-plugin~Name', 'pipelines-plugin~Value'],
```

Comment:
```
    // t('pipelines-plugin~Name') t('pipelines-plugin~Value')
```
This can help us avoid having to bring `t` into the utility and still maintain the fields in the json translation file.

Reply:
Thanks. Updated accordingly